sql/core/src/test/resources/sql-tests/inputs/subquery/exists-subquery/exists-count-bug.sql

```diff
@@ -0,0 +1,21 @@
+create temporary view t1(c1, c2) as values (0, 1), (1, 2);
+create temporary view t2(c1, c2) as values (0, 2), (0, 3);
+create temporary view t3(c1, c2) as values (0, 3), (1, 4), (2, 5);
+
+select * from t1 where exists (select count(*) from t2 where t2.c1 = t1.c1);
```

Comment:
are results before and after this PR the same for these queries?

Reply:
before this PR, these queries failed because aggregations were not allowed in correlated EXISTS/IN subqueries

Reply:
These queries were runnable before - I just checked on master. They returned wrong results due to the COUNT bug.

Reply:
Added tests for the wrong results